r/README.md

```diff
@@ -67,11 +67,6 @@ more complete installation, set the environment variable
 `NOT_CRAN=true`. See `vignette("install", package = "arrow")` for
 details.
 
-For Windows users of R 3.6 and earlier, note that support for AWS S3 is not
```

Comment:
This should probably say that 9.0.0 is the last version that supports Windows R < 4.0, and also note that feature set is limited.